apis/kubedb/constants.go

```diff
@@ -1511,22 +1512,24 @@ const (
 	ClickhousePromethues  = 9363
 	ClickHouseRaftPort    = 9234
 
-	ComponentCoOrdinator = "co-ordinator"
-
-	ClickHousePromethusEndpoint           = "/metrics"
-	ClickHouseDataDir                     = "/var/lib/clickhouse"
-	ClickHouseKeeperDataDir               = "/var/lib/clickhouse_keeper"
-	ClickHouseConfigDir                   = "/etc/clickhouse-server/config.d"
-	ClickHouseKeeperConfigDir             = "/etc/clickhouse-keeper"
-	ClickHouseCommonConfigDir             = "/etc/clickhouse-server/conf.d"
-	ClickHouseTempConfigDir               = "/ch-tmp"
-	ClickHouseInternalKeeperTempConfigDir = "/keeper"
-	ClickHouseTempDir                     = "/ch-tmp"
-	ClickHouseKeeperTempDir               = "/ch-tmp"
-	ClickHouseKeeperConfigPath            = "/etc/clickhouse-keeper"
-	ClickHouseUserConfigDir               = "/etc/clickhouse-server/user.d"
-	ClickHouseLogPath                     = "/var/log/clickhouse-server/clickhouse-server.log"
-	ClickHouseErrorLogPath                = "/var/log/clickhouse-server/clickhouse-server.err.log"
+	ComponentCoOrdinator = "coordinator"
+
+	ClickHousePromethusEndpoint              = "/metrics"
+	ClickHouseDataDir                        = "/var/lib/clickhouse"
+	ClickHouseKeeperDataDir                  = "/var/lib/clickhouse_keeper"
+	ClickHouseConfigDir                      = "/etc/clickhouse-server/config.d"
+	ClickHouseKeeperConfigDir                = "/etc/clickhouse-keeper"
+	ClickHouseCommonConfigDir                = "/etc/clickhouse-server/conf.d"
+	ClickHouseTempConfigDir                  = "/ch-tmp"
+	ClickHouseInternalKeeperTempConfigDir    = "/keeper"
+	ClickHouseKeeperConfigPath               = "/etc/clickhouse-keeper"
+	ClickHouseUserConfigDir                  = "/etc/clickhouse-server/user.d"
+	ClickHouseLogPath                        = "/var/log/clickhouse-server/clickhouse-server.log"
+	ClickHouseErrorLogPath                   = "/var/log/clickhouse-server/clickhouse-server.err.log"
+	ClickHouseClientCertVolumeName           = "cacerts"
+	ClickHouseClientCertVolumeMountPath      = "/etc/clickhouse-server/certs"
+	ClickHouseTempClientCertVolumeName       = "cacerts-temp"
+	ClickHouseTempmClientCertVolumeMountPath = "/ch-certs"
```

Comment:
I told to declare the consts like these
ClickHouseClientCertMountPath = "/etc/clickhouse-server/certs"
ClickHouseClientCertVolumeName = "certs"
ClickHouseTempClientCertVolumeName = "certs-tmp"
ClickHouseTempClientCertMountPath = "/certs-tmp"
ClickHouseTempConfigDir                  = "/config-tmp"